core/qthelper.cpp

```diff
@@ -1414,12 +1414,12 @@ bool diveContainsText(const struct dive *d, const QString &filterstring, Qt::Cas
 	if (!d)
 		return false;
 	return contains(get_dive_location(d), filterstring, cs) ||
-	       (d->divetrip && contains(d->divetrip->location, filterstring, cs)) ||
-	       contains(d->buddy, filterstring, cs) ||
-	       contains(d->divemaster, filterstring, cs) ||
-	       contains(d->suit, filterstring, cs) ||
-	       get_taglist_string(d->tag_list).contains(filterstring, cs) ||
-	       (includeNotes && contains(d->notes, filterstring, cs));
+		(d->divetrip && contains(d->divetrip->location, filterstring, cs)) ||
+		contains(d->buddy, filterstring, cs) ||
+		contains(d->divemaster, filterstring, cs) ||
+		contains(d->suit, filterstring, cs) ||
+		get_taglist_string(d->tag_list).contains(filterstring, cs) ||
+		(includeNotes && contains(d->notes, filterstring, cs));
```

Comment:
again, these are all continuation lines, lined up with the `contains` in the line above.